sympy/sets/sets.py

```diff
@@ -55,11 +55,12 @@ class Set(Basic):
     is_ProductSet = False
     is_Union = False
     is_Intersection = None
-    is_EmptySet = None
     is_UniversalSet = None
     is_Complement = None
     is_ComplexRegion = False
 
+    is_empty = None
```

Comment:
This could probably be `False`. Those places where it would have to be redefined are relatively few.

Reply:
In practice most sets that are not `EmpytSet` will indeed be False, but I think None is the safer choice. E.g. I didn't touch `Intersection` and it's still correct. As I see it there are quite some cases that could (theoretically) turn out to be unrecognized empty sets (Intersection, ProductSet, Complement). `None` is also safer if a new Set subclass is added later on and this property is forgotten / the default not overridden.